jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/datadisplay/Lists.java

```diff
@@ -34,13 +36,11 @@ public Boolean isUnchecked() {
     }
 
     @JDIAction("Is '{name} disabled")
-    @Override
```

Comment:
Do we really don't need override here? The same for line 43

Reply:
Yes, we dont need, because there is nothing to override